container_main.py

```
@@ -72,13 +72,24 @@ def run_api_server():
+        # Hardened gunicorn launch
+        cmd = [
+            "gunicorn",
+            "--bind",
+            f"0.0.0.0:{port}",
+            "--workers",
+            "2",
+            "--timeout",
+            "120",
+            "api_server:app",
+        ]
+        subprocess.run(cmd, check=True)
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)